crates/oxc_transformer/src/commonjs/utils/import.rs

```diff
@@ -0,0 +1,106 @@
+use oxc_ast::ast::{
+    BindingPattern, BindingRestElement, Expression, PropertyKey, Statement, TSTypeAnnotation,
+    TSTypeParameterInstantiation, VariableDeclarationKind,
+};
+use oxc_ast::AstBuilder;
+use oxc_span::SPAN;
+
+pub fn create_require<'a>(target: &str, builder: &'a AstBuilder) -> Expression<'a> {
+    builder.expression_call(
+        SPAN,
+        builder.expression_identifier_reference(SPAN, "require"),
+        None::<TSTypeParameterInstantiation>,
+        builder.vec1(builder.argument_expression(builder.expression_string_literal(SPAN, target))),
+        false,
+    )
+}
+
+/// Generate the `require` bond for a given target.
+/// e.g. for `fs`:
+/// ```js
+/// require('fs')
+/// ```
+///
+/// It can be transformed from:
+///
+/// ```js
+/// import 'fs'
+/// ```
+pub fn create_empty_require<'a>(target: &str, builder: &'a AstBuilder) -> Statement<'a> {
+    builder.statement_expression(SPAN, create_require(target, builder))
+}
+
+/// Generate a namespaced `require` bond for a given target and assignee.
+/// e.g. for `fs`:
+/// ```js
+/// const fs = require('fs')
+/// ```
+///
+/// It can be transformed from:
+///
+/// ```js
+/// import * as fs from 'fs'
```

Comment:
Here also should be ` import * as fs from 'fs'` -> ` const fs = require('fs')`

Reply:
It is a typo... 😂